site/docs/components/avatar/accessibility.mdx

```diff
@@ -0,0 +1,14 @@
+---
+title: Avatar
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Best practices
+
+Decorative avatars (avatars that repeat the information already expressed by the accompanied text) should be hidden from the screen reader to avoid repetition. If you are adding an avatar for decorative purposes, pass `aria-hidden=”true”` to hide its label from the screen reader.
```

Comment:
```suggestion
Decorative Avatars (Avatars that repeat the information already expressed by the accompanied text) should be hidden from the screen reader to avoid repetition. If you are adding an Avatar for decorative purposes, pass `aria-hidden=”true”` to hide its label from the screen reader.
```